modules/openy_gc_personal_training/openy_gc_personal_training.install

```diff
@@ -24,3 +27,77 @@ function openy_gc_personal_training_install() {
     }
   }
 }
+
+/**
+ * Introduce the Personal training type.
+ */
+function openy_gc_personal_training_update_8001() {
+  \Drupal::entityDefinitionUpdateManager()
+    ->installEntityType(new ConfigEntityType([
+      'id' => 'personal_training_type',
+      'label' => new TranslatableMarkup('Personal training type'),
+      'admin_permission' => 'administer gated content configuration',
+      'config_prefix' => 'type',
+      'bundle_of' => 'personal_training',
+      'entity_keys' => [
+        'id' => 'id',
+        'label' => 'label',
+      ],
+      'config_export' => [
+        'id',
+        'label',
+      ],
+    ]));
+}
+
+/**
+ * Import types, fields and forms configs.
+ */
+function openy_gc_personal_training_update_8002() {
+  \Drupal::service('module_installer')->install(['openy_node_session']);
```

Comment:
Do we still need to install openy_node_session after migration to date_recur?

Reply:
Omg! Thx for noticing
Will remove this